extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -279,6 +279,64 @@ describes.repeated('', {
               });
         });
       });
+
+      it('should set html bypassing mustache rendering', () => {
+        sandbox.spy(xhrUtils, 'setupInput');
+        sandbox.spy(xhrUtils, 'setupAMPCors');
+        sandbox.stub(xhrUtils, 'fromStructuredCloneable');
+        sandbox.stub(xhrUtils, 'verifyAmpCORSHeaders');
+
+        return getSsrAmpFormPromise.then(ampForm => {
+          const form = ampForm.form_;
+          const template = createElement('template');
+          template.setAttribute('type', 'amp-mustache');
+          template.content.appendChild(createTextNode('Some {{template}}'));
+          form.id = 'registration';
+          const event = {
+            stopImmediatePropagation: sandbox.spy(),
+            target: form,
+            preventDefault: sandbox.spy(),
+          };
+          const successTemplateContainer = createElement('div');
+          successTemplateContainer.setAttribute('submit-success', '');
+          successTemplateContainer.appendChild(template);
+
+          form.appendChild(successTemplateContainer);
+
+          form.xhrAction_ = 'https://www.xhr-action.org';
+
+          sandbox.stub(ampForm.viewer_, 'sendMessageAwaitResponse')
+              .returns(
+                  Promise.resolve({
+                    data: '<div>much success</div>',
+                  }));
+          const renderedTemplate = createElement('div');
+          renderedTemplate.innerText = 'much success';
+          sandbox.stub(ampForm.ssrTemplateHelper_.templates_, 'findTemplate')
+              .returns(template);
+          sandbox.stub(ampForm.templates_, 'findAndSetHtmlForTemplate')
+              .returns(Promise.resolve({html: renderedTemplate}));
+          sandbox.stub(ampForm.templates_, 'findAndRenderTemplate');
```

Comment:
We shouldn't be testing behavior of another class (`SsrTemplateHelper`). 

We should instead verify that the new function is called, and then add a unit test in test-ssr-template-helper.js.